server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewAssignmentServicesImpl.java

```diff
@@ -47,6 +45,29 @@ public ReviewAssignment save(ReviewAssignment reviewAssignment) {
         return newAssignment;
     }
 
+    @Override
+    public List<ReviewAssignment> saveAll(UUID reviewPeriodId, List<ReviewAssignment> reviewAssignments, Boolean deleteExisting) {
+
+        if(Boolean.TRUE.equals(deleteExisting)) {
+            LOG.warn(String.format("Deleting all review assignments for review period %s", reviewPeriodId));
```

Comment:
Should this use `LOG.info` instead since its not describing a potential problem?